scorecards-site/content/home.md

```diff
@@ -309,7 +309,7 @@ Security Scorecards helps to make informed decisions about security risks and vu
 
 <div>
 
-![endor](assets/logos/endor.svg)
+![endor](assets/logos/endor.png)
```

Comment:
Leave this as `.svg` file. Some internal magic converts the `.png` to `.svg` and puts it here. But it doesn't work if you provide `.svg` directly.

Reply:
Ok, I tried that looks like it is still not rendering. Anything else am I missing? If not who else can help with this?

Reply:
I miss guided you. The real solution was to resize the original SVG file to be smaller and move it from `assets/logos` to `static/assets/logos`. Hopefully, this should work now.


>  If not, who else can help with this?

At this point, it's just us (Scorecard maintainers) :) We might need a bit of trial and error to get a hang of this. I'll try to write up documentation for this as I learn stuff.

Reply:
Looks good to submit now.